### ISSUE TITLE
Notes on using celltypist 

### ISSUE THREAD

**Issue by emdann** [Teichlab/Pan_fetal_immune#9]
From meeting with Cecilia
- Chuan is adding new features (incl. use from anndata objects rather than .csv files), so the version in pypi `celltypist-dev` is out of date. Trained models are on the farm and we could pickle them from there, see [example notebook](https://github.com/emdann/Pan_fetal_immune/blob/working_emma/legacy_code/Z01-1_20200930_covid-19.html) from Rik
- Some of our samples are also in the training set: check [here](https://docs.google.com/spreadsheets/d/1ON8ubTSaR5Ffsz_oLcRRaspKVxXh_qIE/edit#gid=387962573) to avoid overfitting issues and use to translate our uniform labels to celltypist vocabulary
- Significant improvements from scaling gene expr profiles
- Some high level annotation (e.g. "T cell") is present also in deeper hierarchy: that is because some cells don't have deep labels in original annotation
